A_perfect_dish_to_share.py

```diff
@@ -17,8 +30,6 @@ def is_perfect(number_to_check):
             dividers_sum = dividers_sum + divider + number_to_check / divider
         divider += 1
 
-    # If sum of divisors is equal to
-    # n, then n is a perfect number
     return True if dividers_sum == number_to_check else False
```

Comment:
try to rewrite the return line in a more elegant manner?

Reply:
What is meant by a more elegant manner? use a Boolean variable?

Reply:
Think I understand, in the original code there were 2 conditions so I used this return, I will fix it.